databricks_cli/configure/config.py

```diff
@@ -37,11 +38,15 @@ def provide_api_client(function):
     """
     @six.wraps(function)
     def decorator(*args, **kwargs):
+        ctx = click.get_current_context()
+        command_name = "-".join(ctx.command_path.split(" ")[1:])
```

Comment:
some commands may have "-" in it. What would be the standard way of doing it? How about use `"/".join` instead?

Reply:
Actually, would it be cleaner if you just add the command and sub command separately? Since we already have a json, it would be good to keep the structured data.

Reply:
Hm command and subcommand stored separately may work, but I'm worried that there may be even more nested subcommands in the future (ie. a subcommand within a subcommand) and that will be inconvenient to handle

Reply:
Actually I think it may be appropriate just to include the spaces

Reply:
This function is a little confusing. Could you add some examples in comments?

Reply:
@xuanwang14 @alinxie currently it's hard to analyse the usage of subcommands.